### ISSUE TITLE
Default Shortcut not working on Ubuntu 22.04

### ISSUE THREAD

**Issue by Detrua** [kumakichi/Deepl-linux-electron#9]
In Ubuntu 22.04 Ctr+Alt+D you hide all normal windows.
After starting the app manually and changing the Shortcut it's working perfectly! Thank you

**Comment by koloved** [comment]
same

**Comment by kumakichi** [comment]
Yes, the pre-defined shortcut Ctrl+Alt+D is conflict with the default ubuntu system shortcut to hide all windows (Settings -> Keyboard -> Keyboard Shortcuts -> Navigation -> Hide all normal windows)

Considering other distributions, choosing a combination that does not conflict with the system's default shortcut keys is difficult, so it is better to leave it to the user to configure a shortcut combination that suits them in the way described in the README

**Comment by vadi2** [comment]
Does setting any other shortcut work? I have tried `ctrl + . `, made sure nothing else is using it, but the application is still not coming up in Ubuntu 24.04 with Wayland.
